integration/query_test.go

```diff
@@ -834,3 +838,640 @@ func TestQueryCommandSingleBatch(t *testing.T) {
 		})
 	}
 }
+
+func TestQueryBatchSize(t *testing.T) {
```

Comment:
Feel free to remove all changes in `query_test.go` file. They were moved and renamed to `getmore_test.go` :hugs:

Reply:
Done